### PR TITLE
move project run/setup to devstack

### PR DESCRIPTION
**Pull request by Zacharis278**
~This depends on https://github.com/edx/devstack/pull/379.   Without it there is no way to actually run the project.~
Edit: This is now in master

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,35 +1,12 @@
-shell: ## run a shell on the cookie-cutter container
-	docker exec -it edx.gradebook /bin/bash
+shell: ## run a shell
```
Comment (iloveagent57) [inline Makefile]: Will I be able to do `make gradebook-shell` from inside devstack now?

File: README.md [hunk 2]
```diff
@@ -23,18 +23,18 @@ npm i --save @edx/gradebook
 
 After cloning the repository, run `make up-detached` in the `gradebook` directory - this will build and start the `gradebook` web application in a docker container.
```
Comment (iloveagent57) [inline README.md]: The `up-detached` command was deleted, so this line should be updated.

Comment (Zacharis278) [reply README.md]: good call

File: Makefile (hunk 1 above)
Comment (Zacharis278) [reply Makefile]: Yeah you can get to shell via devstack as well I just left it as a shortcut.  Ultimately, it's unnecessary.  However there's no equivalent of the npm install shortcut in devstack so we probably want to keep that.

**Comment by Zacharis278** [comment]
@iloveagent57 I've made the requested updates and removed all docker references from the makefile

**Pull request merged**